diff --git a/src/components/SearchFilter.vue b/src/components/SearchFilter.vue
index a815202..688d8da 100644
--- a/src/components/SearchFilter.vue
+++ b/src/components/SearchFilter.vue
@@ -149,7 +149,7 @@
         <b-button v-if="canShowExampleCode" type="button" variant="secondary" @click="showCodeModal = true">{{ $t('exampleCode.title') }}</b-button>
       </b-card-footer>
     </b-card>
-    <b-modal v-if="canShowExampleCode" v-model="showCodeModal" :title="$t('exampleCode.title')" size="xl">
+    <b-modal v-if="canShowExampleCode" v-model="showCodeModal" :title="$t('exampleCode.title')" size="xl" teleport-to="#stac-browser">
       <SearchCode
         v-if="showCodeModal"
         :searchLinks="codeExampleSearchLinks"
